crates/cext/src/transpiler/transpile_layout.rs

```diff
@@ -0,0 +1,337 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use crate::pointers::const_ptr_as_ref;
+use qiskit_transpiler::transpile_layout::TranspileLayout;
+
+/// @ingroup QkTranspileLayout
+/// Return the number of qubits in the input circuit to the transpiler.
+///
+/// @param layout A pointer to the ``QkTranspileLayout``.
```

Comment:
These first 2 functions miss the `@return` doc 🙂

Reply:
Added in: https://github.com/Qiskit/qiskit/pull/14780/commits/c12428039a91b0abd1d180d70646b8793f9722df